safe/gis/vector/multi_buffering.py

```diff
@@ -60,12 +60,17 @@ def multi_buffering(layer, radii, callback=None):
     input_crs = layer.crs()
     feature_count = layer.featureCount()
 
-    # Set the new hazard value field
+    # Set the new hazard class field.
     fields = layer.fields()
     field_type = hazard_class_field['type']
```

Comment:
Can you update the code to use this helper for these two fields ? https://github.com/inasafe/inasafe/blob/develop/safe/gis/vector/tools.py#L211